packages/vincent-tools/vincent-tool-uniswap-swap/.env.test.example

```diff
@@ -13,6 +18,6 @@ TEST_APP_DELEGATEE_PRIVATE_KEY=
 # Test Agent Wallet PKP Owner Address: 
 TEST_AGENT_WALLET_PKP_OWNER_PRIVATE_KEY=
 
-ERC20_APPROVAL_TOOL_IPFS_ID=QmWavjbn2fUkgnEV52ExhZN4CikfzHJZ5GW13HSn9Tu2M9
-UNISWAP_SWAP_TOOL_IPFS_ID=QmZnXmSvJCj1dG8MtP8m2riVttyV8Ed7GQSrtV6z3komWE
-SPENDING_LIMIT_POLICY_IPFS_ID=QmYB2zcP2jSHyGVP3S9pwtmgtQY6BinZ9DFCkyp6CsdsNX
+ERC20_APPROVAL_TOOL_IPFS_ID=
+UNISWAP_SWAP_TOOL_IPFS_ID=
+SPENDING_LIMIT_POLICY_IPFS_ID=
```

Comment:
Should these be removed from the .env file entirely at this point?

Reply:
Oh because we should be importing them from the packages - didn't realize that. Will test and remove